packages/manager/apps/pci-kubernetes/src/pages/new/steps/node-pool/NodePoolAntiAffinity.component.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsText,
+  OsdsFormField,
+  OsdsToggle,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { ANTI_AFFINITY_MAX_NODES } from '@/constants';
+
+type NodePoolAntiAffinityProps = {
+  isChecked: boolean;
+  isEnabled: boolean;
```

Comment:
```suggestion
  checked: boolean;
  enabled: boolean;
```
Personally I prefer the react syntax instead of ODS one